app/controllers/application_controller.rb

```diff
@@ -20,28 +19,52 @@ def jsonapi_render(collection, filters)
   end
 
   def set_user
-    @current_user = if auth_token.present?
-        User.from_jwt(Authenticator.from_token(auth_token))
-      else
-        nil
-      end
+    return nil unless auth_token.present?
+
+    @current_user = User.where(
+                      id: client.authenticated_user_id,
```

Comment:
OOI - what is the PK on the users table? does assigning an external id from the main API cause issues here?

Reply:
What kind of issues? I wanted to keep the ids the same as panoptes for projects, users, and workflows to minimize confusion. I don't see a reason not to, unless you do?

Reply:
I was wondering how default primary key columns handle value insertion(s) in this column and how this impacts on the PK id sequence etc. I dare say it's fine, just wondering if you've considered this.

Reply:
I believe that it's fine, a few of our other apps (Parrish ones mostly) do this so that assumptions can be made based on Panoptes resource ids. `id` is still the primary key (indexed), and while it seems like it used to be an attr_protected (requiring a workaround to set on create), that's no longer the case.